charts/kong/README.md

```diff
@@ -300,6 +300,11 @@ individual services: see values.yaml for their individual default values.
 * `manager`
 * `portal`
 * `portalapi`
+* `status`
+
+Unlike most services, `status` cannot be exposed outside the pod, and cannot
+create a Kubernetes service or ingress. It supports the settings under
+`SVC.http` and `SVC.tls` only.
```

Comment:
> Unlike most services

Unlike all other services

> cannot be exposed outside the pod, and cannot
create a Kubernetes service or ingress

Can we instead frame this as "The chart doesn't allow exposing the endpoint as outside the Kubernetes cluster because the endpoint is meant for intra-cluster concerns only.".

Reply:
There's no service creation either--it's not accessible outside the pod, barring things like readiness/liveness checks. Added the why in front, but I think we should keep the specifics of what it does--K8S-saavy users should recognize what no service/no ingress means in practice.